core/players-handbook-2024/archetypes/paladin-glory.xml

```diff
@@ -0,0 +1,167 @@
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_PALADIN_24_OOG_INSPIRING_SMITE</requirements>
+		<description>
+			<p>Immediately after you cast <i>Divine Smite</i> you can expend one use of your Channel Divinty and distribute Temporary Hit Points to creatures of your choice within 30 feet of yourself, which can include you.The total number of Temporary Hit Points equals 2d8 plus your Paladin level, divided among the chosen creatures however you like.</p>
+		</description>
+		<sheet alt="Inspiring Smite" usage="Channel Divinity">
+			<description>Immediately after you cast Divine Smite you can distribute up to 2d8+{{level:paladin}} Temporary HP to creatures of your choice within 30ft of yourself, which can include you, divided however you like.</description>
+		</sheet>
+	</element>
+	<element name="Level 3: Oath Spells" type="Archetype Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_OOG_OATH_SPELLS">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_PALADIN_24_OOG_OATH_SPELLS</requirements>
+		<description>
+			<p>The magic of your oath ensures you always have certain spells ready; when you reach a Paladin level specified in the Oath of Glory Spells table, you thereafter always have the listed spells prepared.</p>
+			<h5 class="caption">Oath of Glory Spells</h5>
+			<table>
+				<thead>
+					<tr><td>Paladin Level</td><td>Spells</td></tr>
+				</thead>
+				<tr><td>3rd</td><td><em>Guiding Bolt, Heroism</em></td></tr>
+				<tr><td>5th</td><td><em>Enhance Ability, Magic Weapon</em></td></tr>
+				<tr><td>9th</td><td><em>Haste, Protection from Energy</em></td></tr>
+				<tr><td>13th</td><td><em>Compulsion, Freedom of Movement</em></td></tr>
+				<tr><td>17th</td><td><em>Legend Lore, Yolande's Regal Presence</em></td></tr>
+			</table>
+		</description>
+		<sheet display="false" />
+		<rules> <!--
+			<grant type="Spell" id="ID_PHB_SPELL_GUIDING_BOLT" spellcasting="Paladin" level="3" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_HEROISM" spellcasting="Paladin" level="3" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_ENHANCE_ABILITY" spellcasting="Paladin" level="5" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_MAGIC_WEAPON" spellcasting="Paladin" level="5" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_HASTE" spellcasting="Paladin" level="9" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_PROTECTION_FROM_ENERGY" spellcasting="Paladin" level="9" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_COMPULSION" spellcasting="Paladin" level="13" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_FREEDOM_OF_MOVEMENT" spellcasting="Paladin" level="13" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_COMMUNE" spellcasting="Paladin" level="17" prepared="true" />
+			<grant type="Spell" id="ID_PHB_SPELL_FLAME_STRIKE" spellcasting="Paladin" level="17" prepared="true" /> -->
```

Comment:
Wrong spells listed here.

Reply:
you mean the commented out grants? that's not really an issue.. they were going to be fixed once spells are out..

Reply:
No, I meant the 5th level spells granted at lvl 17 don't match the table. Grants are giving _Commune_ and _Flame Strike_, but should be _Legend Lore_ and _Yolande's Regal Presence_